README.md

```diff
@@ -39,9 +39,11 @@ else!
 
 ## Features
 
-* Support for signing Python package distributions using an OpenID Connect identity
-* Support for publishing signatures to a [Rekor](https://github.com/sigstore/rekor) instance
-* Support for verifying signatures on Python package distributions
+* Support for keyless Sigstore signature generation and verification,
+  [a la `cosign`](https://docs.sigstore.dev/cosign/openid_signing/)
+* Support for ["ambient" OpenID Connect identities](#signing-with-ambient-credentials)
+* A comprehensive [`sigstore` CLI](#usage) and corresponding
+  [Python API](https://sigstore.github.io/sigstore-python), for programmatic usage
```

Comment:
```suggestion
* Support for keyless signature generation and verification with [Sigstore](https://www.sigstore.dev/)
* Support for signing with ["ambient" OpenID Connect identities](https://github.com/sigstore/sigstore-python#signing-with-ambient-credentials)
* A comprehensive [CLI](https://github.com/sigstore/sigstore-python#usage) and corresponding
  [importable Python API](https://sigstore.github.io/sigstore-python)
```
Some tweaks. The anchor links will unfortunately not work on the PyPI project page, so those need to be full links.